frontend/cypress/e2e/map.cy.js

```diff
@@ -64,3 +64,49 @@ describe('Analysis should work for Livelihoods zones layer', () => {
     cy.get('#analysisPanelView').should('be.visible');
   });
 });
+
+describe('Share modal should show shorten URL', () => {
+  const shortenUrl = 'http://localhost:3000/share/cb69ec73745eb70c4a5d';
```

Comment:
Maybe you can construct this URL by importing the fixture (`share-url-encode.json`)?

Reply:
you are right